api/app/main.py

```diff
@@ -176,6 +177,11 @@ def get_acled_incidents(request: Request):
     return Response(content=response.content, media_type="application/json")
 
 
+@app.get("/hdc")
+def wrap_get_hdc_stats(request: Request):
```

Comment:
you need to better type this request

Reply:
@reed1 can you take care of this so that we can merge? Thanks.

Reply:
You need to type the request, like the one for kobo/dates. This will allow the API to parse the request and let the user know what parameters are missing, if any. While you're at it, maybe you can type the ACLED request as well? Let me know if you run into any issues

```
@app.get("/kobo/dates")
def get_kobo_form_dates(
    koboUrl: HttpUrl,
    formId: str,
    datetimeField: str,
    filters: Optional[str] = None,
):
```

Reply:
@ericboucher , updated, let me know if that works for you. I tried to use Depends to avoid duplicating function signature. It works on POST, but not GET, only on recent version of fastapi it will work on GET.